modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/Query/Linq/CacheLinqTest.Strings.cs

```diff
@@ -110,6 +110,30 @@ public void TestStrings()
 
             // String + int
             CheckFunc(x => x + 10, strings);
+
+            // string.Compare(string strA, string strB)
+            CheckWhereFunc(x => string.Compare(x, "Person_1300") < 0, strings);
```

Comment:
What about `>=` and `<=`, do we support that?

Reply:
agree, added tests for `<=` and `>=`